### PR TITLE
[#55950] updated documentation for storage health checks

### PR DESCRIPTION
**Pull request by Kharonus**
[#55950](https://community.openproject.org/work_packages/55950)

### WAT?

- updated text and overall structure
- missing current screenshots

### Hint
- @MayaBerd we need to talk about some headings, the future of this page and the screenshots, once you are back.

### CODE REVIEW

File: docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md [hunk 1]
```diff
@@ -8,29 +8,52 @@ keywords: file storages, Nextcloud error, Nextcloud troubleshooting, OneDrive er
...
+| ERR_UNKNOWN            | An unknown error occurred.                                                            | There can be multiple reasons and the error source was not foreseen.                                                                                                                                      | Errors of this kind are logged to the server logs. Look for a log entry starting with `Connection validation failed with unknown error:`                                                                                                                                                                                                                                       |
+
+#### Nextcloud compatibility
```
Comment (dominic-braeunlein) [inline docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md]: @Kharonus 
I am wondering about the reason of having a own section for "Nextcloud compatibility"?

**Review by dominic-braeunlein — APPROVED** [review]

File: docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md (hunk 1 above)
Comment (Kharonus) [reply docs/system-admin-guide/files/external-file-storages/file-storage-troubleshooting/README.md]: As just clarified, the heading might be misleading. I tried to have here a placeholder, that explains to the reader, that this is not available for nc file storages (yet). Once the feature is done, this section will be deleted, and the rest of the text updated.

**Review by MayaBerd — APPROVED** [review]
thanks @Kharonus, i've committed my changes

**Comment by Kharonus** [comment]
I am satisfied with the new screenshots and changes by @MayaBerd , tusen takk!

what about you, @dominic-braeunlein ?

**Review by dominic-braeunlein — APPROVED** [review]
LGTM 👍

**Pull request merged**